src/main/webapp/VAADIN/themes/mytheme/mytheme.scss

```diff
@@ -44,8 +43,11 @@
     width: 300px;
     box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
     transition: all 0.3s cubic-bezier(.25,.8,.25,1);
+  }
+  .card.journal {
```

Comment:
Why readd this?

Reply:
Need further explanation from you. I don't see what you mean